### PR TITLE
Trying to kick this off again

### PR DESCRIPTION
**Pull request by yatli**
Hi @baronfel !

Guess things are even rougher back then, but it's getting better.

### CODE REVIEW

**Comment by baronfel** [comment]
Wow, nice work iterating on it :) Would you like to have this merged in as a base and then contribute more work iteratively? Alternatively we could move this over to fsprojects and I could make you a maintainer as well.

**Comment by yatli** [comment]
I'll be glad to drive this now and then as a maintainer -- structural editing for F# is going to be cool.
As long as there's a repo, people can see it, and we can discuss stuff in threads/PRs I can get it going.
Moving to fsprojects would be great, but at this moment it's just baby steps -- what do you prefer?

**Comment by baronfel** [comment]
I'll just add you as a maintainer here while it's still growing rapidly, but when it gets close I think a move to fsprojects for polishing/docs/a permanent home makes sense :)

**Comment by yatli** [comment]
Yeah, a "keyframe" is when it gets useful enough, and we figure out how to plug this into editors like `nvim-tree-sitter` -- it's going to need a permanent home so people can download it with something like `:TSInstall fsprojects/tree-sitter-fsharp`

File: corpus/strings.txt [hunk 1]
```diff
@@ -8,17 +8,42 @@ simple string
...
+---
+
+(test (string))
+
 ===
 simple char
 ===
 
-'I'
+' '
```
Comment (yatli) [inline corpus/strings.txt:26]: This is still not working 😂

**Comment by baronfel** [comment]
I merged this and I've sent you an invite as a collaborator for subsequent things.

**Comment by yatli** [comment]
Thanks!

**Pull request merged**